lib/extensions/requirement_block_macro.rb

```diff
@@ -26,6 +26,7 @@ def trim(s)
 end
 
 adoc_files = Dir.glob("**/*.adoc")
+adoc_files.sort!
```

Comment:
Travis build fails due to OSX / Linux differences. `dir.glob` returns differently-ordered results depending on the filename. Adding this workaround here to sort the files first before diving in.